packages/framer-motion/src/animation/waapi/easing.ts

```diff
@@ -1,4 +1,12 @@
-import { BezierDefinition, EasingDefinition } from "../../easing/types"
+import { BezierDefinition, Easing, EasingDefinition } from "../../easing/types"
+
+export function isEasingSupported(easing?: Easing | Easing[]) {
+    return (
+        !easing ||
+        Array.isArray(easing) ||
+        (typeof easing === "string" && supportedWaapiEasing[easing])
```

Comment:
This is pretty weird logic to me.
Why is easing always supported if `easing` is an array or easing is falsy?

Reply:
Easing is falsy === default array
Easing is array === bezier curve

Reply:
Not 100% sure, but maybe we should add that as a comment. Not feeling strongly though, so do what you prefer